openqabot/openqa.py

```diff
@@ -20,15 +21,24 @@ def __bool__(self) -> bool:
         return self.url.netloc == "openqa.suse.de"
 
     def post_job(self, settings) -> None:
-        logger.info("openqa-cli api --host %s -X post isos %s" % (self.url.geturl(), ' '.join(['%s=%s' % (k, v) for k,v in settings.items()])))
+        logger.info(
+            "openqa-cli api --host %s -X post isos %s"
+            % (
+                self.url.geturl(),
+                " ".join(["%s=%s" % (k, v) for k, v in settings.items()]),
+            )
+        )
         try:
             self.openqa.openqa_request("POST", "isos", data=settings, retries=3)
+        except RequestError as e:
+            logger.error("openQA returned %s" % e[-1])
+            logger.error("Post failed with {}".format(pformat(settings)))
         except Exception as e:
             logger.exception(e)
             logger.error("Post failed with {}".format(pformat(settings)))
 
     def get_jobs(self, data: Data):
-        logger.debug("getting openqa for %s" % pformat(data))
+        logger.info("Getting openqa for %s" % pformat(data))
```

Comment:
```suggestion
        logger.info("Getting openQA for %s" % pformat(data))
```
Same counts for other occurrences. About this specific one: Should it actually be "openQA job"?
```suggestion
        logger.info("Getting openQA job for %s" % pformat(data))
```